firebase-firestore/src/main/java/com/google/firebase/firestore/util/Function.java

```diff
@@ -17,7 +17,10 @@
 import javax.annotation.Nullable;
 
 /**
- * Determines an output value based on an input value; a pre-Java-8 version of {@link
+ * This is a copy of
```

Comment:
Please move this outside the class JavaDoc, as you likely don't want this to be the summary of this API. I would recommend just moving this above in `//` style comment.

Reply:
Done.